.gitignore

```diff
@@ -191,6 +191,7 @@ node_modules/
 /ajax/libs/jquery-easing/1.3.0/
 /ajax/libs/jquery-easing/1.3.1/
 /ajax/libs/libphonenumber/libphonenumber*/
+/ajax/libs/slippry/v.1.2/
```

Comment:
@extend1994 Sorry about my mistake, I use old version of `autoupdate` to fetch the version.
I think this PR is fine now, could you rebase this branch with our latest master? Thanks.